testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/EnvSinglenodeOauth2HttpsProxy.java

```diff
@@ -61,8 +61,8 @@ public void extendEnvironment(Environment.Builder builder)
                             forHostPath(configDir.getPath("config.properties")),
```

Comment:
Do we still need `SeleniumChrome` extender?

Reply:
No, we don't removed.